core/src/main/scala/com/ing/baker/java_api/JBaker.scala

```diff
@@ -91,10 +91,8 @@ class JBaker private (jRecipe: JRecipe,
     *
     * @param processId the identifier of the process
     * @param event The event to fire
-    * @throws BakerException
     * @return
     */
-  @throws(classOf[BakerException])
```

Comment:
Those are all runtime exceptions.
We only need to supply all checked exceptions and only for the classes directly used in the Java code.